### PR TITLE
AMP action target and goBack method

### LINKED ISSUE #5225: Proper `goback` functionality in AMP
**UPDATE** [May 14 2020]
I'm pretty sure that the AMP team will spent another 5 years on decision "TO DO vs NOT TO DO". The workaround is simple - use iframe with JS code inside which calls `history.back`. Yes it's ugly but it works.

----

Can you please add some param for `<button>` or `<a>` tags which will do javascript:window.history.back() call?

I use amp-iframe as workaround for this issue, but it will be much more convenient to have dedicated "amp-" tag for it or an attribute for `<button>` or `<a>` tags.

Thank you in advance

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #5225.

/cc @bpaduch for spec/doc changes.

### CODE REVIEW

File: spec/amp-actions-and-events.md [hunk 1]
```diff
@@ -143,3 +148,20 @@ For example, the following is possible in AMP.
     <td>Hides the referenced user notification element</td>
   </tr>
 </table>
+
+
+## `AMP` target
```
Comment (cramforce) [inline spec/amp-actions-and-events.md]: Why uppercase?

Comment (dvoytenko) [inline spec/amp-actions-and-events.md]: That's the agreement we came up with between this action and binder. It's kind of a unique target: not supported by any element. Capital case also seemed to have a lower chance of collision with existing IDs.

File: test/functional/test-standard-actions.js [hunk 2]
```diff
@@ -55,13 +56,35 @@ describes.sandboxed('StandardActions', {}, () => {
...
+  describe('"AMP" global target', () => {
+    it('should implement goBack', () => {
+      const history = window.services.history.obj;
+      const goBackStub = sandbox.stub(history, 'goBack');
+      standardActions.handleAmpTarget({method: 'goBack'});
+      expect(goBackStub).to.be.calledOnce;
+    });
+
+    it('should implement setState', () => {
+      const setStateStub = sandbox.spy();
```
Comment (choumx) [inline test/functional/test-standard-actions.js]: Nit: setStateSpy

File: src/service/standard-actions-impl.js [hunk 3]
```diff
@@ -50,16 +56,38 @@ export class StandardActions {
    * @private
    */
   installActions_(actionService) {
+    actionService.addGlobalTarget('AMP', this.handleAmpTarget.bind(this));
     actionService.addGlobalMethodHandler('hide', this.handleHide.bind(this));
   }
 
   /**
+   * Hanldes global `AMP` actions.
```
Comment (choumx) [inline src/service/standard-actions-impl.js]: Nit: s/Hanldes/Handles

File: test/functional/test-standard-actions.js (hunk 2 above)
Comment (dvoytenko) [inline test/functional/test-standard-actions.js]: Done

File: src/service/standard-actions-impl.js (hunk 3 above)
Comment (dvoytenko) [inline src/service/standard-actions-impl.js]: Done

**Comment by keepitsimple** [comment]
Is it experimental or regular feature? If it's experimental when it will be regular?

**Comment by dvoytenko** [comment]
It's a regular feature. It'll go live in PROD probably after Jan 5th. You can see the progress in https://github.com/ampproject/amphtml/releases

**Pull request merged**